modules/apps/object/object-test/src/testFunctional/paths/ObjectAdmin.path

```diff
@@ -262,7 +262,7 @@
 	</tr>
 	<tr>
 		<td>OBJECT_RELATIONSHIPS_TAB</td>
-		<td>//a[contains(text(),'Relationships')]</td>
+		<td>//a[contains(text(),'Relationship')]</td>
```

Comment:
Hi @andreefarias 

You don't need to change this line, let me know why you changed it.

Thanks!

Reply:
There are test cases that look for "Relationship". I believe it gets more generic this way.

Reply:
Okay.